src/commands/moderation/warn.ts

```diff
@@ -75,10 +67,74 @@ export const warn: CommandDefinition = {
             description: `<@${id}> has been warned.`,
             fields: [
                 { name: 'Reason', value: `${warnReason}`, inline: true },
-                { name: 'Warn Count', value: `${warnCount + 1}`, inline: true },
+                { name: 'Warn Count', value: `${newWarnCount}`, inline: true },
             ],
         });
 
         await message.channel.send({ embeds: [embed] }).catch(console.error);
+
+        // Action prompt
+        const actionEmbed = createEmbed({
+            title: 'Action',
+            description: 'Will there be any action taken against this user?',
```

Comment:
> description: 'Will there be any action taken against this user?',

Would change to: 'Is any further action going to be taken against this user?'

Reply:
"Will any further action be taken against this user" sounds better to me

Reply:
+1